docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>Is it really supposed to say single-Pauli Z <em>errors</em> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
fixed